Model/CreditCardPaymentMethod.php

```diff
@@ -233,4 +233,19 @@ private function _getCurrencyOptions()
 
         return $aOptions;
     }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     *
+     * @since 1.0.0
+     */
+    public function getPublicFieldNames()
```

Comment:
If this is used for the support email that just those fields are shown its wrong. I find it better to just clear the fields that should not be send away.

Reply:
There are two views on the topic. Originally, I used the approach you suggest. In our internal CR we agreed to use this approach as it is more secure when you wite-list the fields you want to show, which cannot result in sending some secure fields by mistake.